packages/babel-preset-expo/package.json

```diff
@@ -47,7 +47,7 @@
     "@babel/plugin-transform-parameters": "^7.22.15",
     "@babel/preset-env": "^7.20.0",
     "@babel/preset-react": "^7.22.15",
-    "@react-native/babel-preset": "^0.73.18",
+    "@react-native/babel-preset": "^0.74.1",
```

Comment:
nix: as we touch the babel-preset version here. we may like to bump the `@react-native/assets-registry` version from expo-asset and expo-image. so that we don't need resolutions anymore.
also `@react-native/dev-middleware` inside @expo/cli